cmd/asset-syncer/main.go

```diff
@@ -22,6 +22,15 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	databaseType     string
+	databaseURL      string
+	databaseName     string
+	databaseUser     string
+	databasePassword string
```

Comment:
From what I can tell below, we could just have:
```
dbConfig database.Config
```
here, and pass that down, rather than handling all 4 separately?